lighthouse-cli/performance-experiment/server.js

```diff
@@ -22,8 +22,10 @@
  * Functionality:
  *    Host experiment.
  *    Report can be access via URL: /?id=[REPORT_ID]
- *    Browser can request lighthousr rerun by sending POST request to URL: /rerun?id=[REPORT_ID]
- *      This will rerun lighthouse with additional cli-flags received from POST request data and
+ *    Browser can request lighthousr rerun by sending POST request to URL:
```

Comment:
plz revert any changes to these js files.
they should not be affected by the clang-format.